CIP-Liesenfelt-Shelleys_Voltaire_decentralization_update/README.md

```diff
@@ -260,7 +260,59 @@ The winners of block forks and slot battles are determined by which pool has a l
 
 ## Leverage-based Pool Ranking Recommendation
 
-Pool ranking scores in wallet browsers have a significant impact on delegator choices. When pledge becomes the most important factor for total pool size lower leverage factors are more desirable. Lists should be sorted by leverage and presented in an ascending order with the lowest leverage pools first.
+Pool ranking scores in wallet browsers have a significant and powerful game theory impact on delegator pool choices, yet it is often overlooked and not transparent in wallets.
+
+When pledge becomes the most important factor for total pool size, lower leverage factors are more desirable. Lists should be sorted by leverage and presented in an descending order with the lowest leverage pools first.
+
+### Ranking Equation
+
+The recommended ranking equation starts with the highest score of 10. The pools are down-ranked solely based on leverage, saturation, and fee factors.
+
+    //equation
+    ranking_score = 10 - max{ leverage_factor, saturation_factor } - fee_factor
+
+    //variables
+    leverage_factor = 10 * (pool_leverage/L)^A
+    saturation_factor = 2 * (pool_stake/(saturation_stake * C))^B
+    fee_factor = D * pool_fee_margin
+
+    //parameters
+    - A is 2.0, has range (0,10.0), can be tweaked
+    - B is 5.0, has range (0,10.0), can be tweaked
+    - C is 0.9, has range (0,2.0), can be tweaked
+    - D is 50, has range (0-100) ish to be harsh, can be tweaked
+    - pool_leverage = delegation / pledge
+    - pool_stake = pledge + delegation
+    - saturation_stake = total_live_stake/k (e.g. 68M₳ "soft-cap upper limit")
+    - pool_fee_margin is in range (0-100)% (fixed fee + margin combined)
+
+To evaluate rank using the current reward scheme:
+
+    //current reward scheme only
+    ranking_score = 10 - max{ leverage_factor , saturation_factor } - fee_factor - fixed_fee_factor
+
+    //variables
+    //if fee = minFee, term drops out
+    //if fee > minFee, term nonzero but loses relevance w/ increased stake
+    //fixed fee matters less to rewards as stake grows, so too here
+    fixed_fee_factor = E * (fee-minFee) / stake 
+
+    //parameters
+    - E is 100, has range (0,100_000), can be tweaked
+    - fee cannot be less than minFee
+
```

Comment:
Removed grading system that was here.  Also simplified a pool's business life-cycle below.